defaults/main.yml

```diff
@@ -45,6 +45,8 @@ php_upload_max_filesize: "64M"
 php_post_max_size: "32M"
 php_date_timezone: "America/Chicago"
 
+php_allow_url_fopen: "Off"
+
```

Comment:
Can you remove this extra line? This can be grouped with other 'misc options' instead of being on its own.